frcnn_detector.py

```diff
@@ -93,10 +94,17 @@ def warm_up(self):
         dummy_image = np.ones((3, self.__resize_width, self.__resize_height)) * 255.0
 
         # prepare the arguments
-        arguments = {
-            self.__model.arguments[self.__args_indices["features"]]: [dummy_image],
-            self.__model.arguments[self.__args_indices["rois"]]: [dummy_rois]
-        }
+        if (self.__isPythonModel):#python model

```

Comment:
Remove the comment at the end of the line. It's pretty clear what _isPythonModel means :)